Upendo.Modules.DnnPageManager/Scripts/common.js

```diff
@@ -0,0 +1,152 @@
+﻿var common = common || {};
+
+common.Utils = function () {
+
+    var get = function (httpMethod, action, service, params, success, fail, always) {
+        var jqxhr = $.ajax({
+            url: "http://localhost.dnnpagemanager/DesktopModules/Upendo.Modules.DnnPageManager/API/Settings" + "/" + action,
```

Comment:
You've hard-coded a domain, but a local domain, and using a non-SSL URL.  This value should be put together using a more dynamic method.  This could be pulling it from the current website URL directly in JS, or by passing values to the UI from the backend code as variables, and then used to concatenate the URL seen here.  The only hard-coded part should be the path between the domain name and the parameters, `/DesktopModules/Upendo.Modules.DnnPageManager/API/Settings`.

Reply:
Hi @WillStrohl . The solution to this is already in commit #5503b65649a8e55d823bc29393b293a592554d7b